src/client/pages/Landing/Landing.stories.tsx

```diff
@@ -7,6 +7,7 @@ import { Landing } from './Landing'
 export default {
   title: 'Landing',
   component: Landing,
+  parameters: { layout: '' },
```

Comment:
Just curious - why is this needed? 🤔

Reply:
Good question! 👍 I've added a global setting for layouts in `preview.js`, and the default is `centered` which is nice for most components unless it's more of a page component. So this resets the layout